templates/cinder/config/00-config.conf

```diff
@@ -23,10 +23,6 @@ osapi_volume_workers = 4
 control_exchange = openstack
 api_paste_config = /etc/cinder/api-paste.ini
 
-# For containers we want to log to stdout instead of the default stderr, as
```

Comment:
We don't need this anymore as a "global" option because we can enforce it per-service